packages/modular-scripts/src/build/buildPackage/maybeCopyRootLicense.ts

```diff
@@ -0,0 +1,24 @@
+import * as fs from 'fs-extra';
+import * as path from 'path';
+import getPrefixedLogger from '../../utils/getPrefixedLogger';
+import getModularRoot from '../../utils/getModularRoot';
+
+export async function maybeCopyRootLicense(
+  target: string,
+  targetOutputDirectory: string,
+) {
+  const logger = getPrefixedLogger(target);
+  const modularRoot = getModularRoot();
+
+  const targetOutputLicense = path.join(targetOutputDirectory, 'LICENSE');
+  if (!fs.existsSync(targetOutputLicense)) {
```

Comment:
Sometimes people use `LICENSE.txt`, `LICENSE.md` etc. Maybe we should copy any file of the form `LICENSE.*`?